lib/video-moq/index.ts

```diff
@@ -210,42 +233,58 @@ export class VideoMoq extends HTMLElement {
 					</button>
 					<ul id="tracklist" class="absolute bottom-6 right-0 mt-2 w-40 rounded bg-black-80 p-0 text-white shadow-lg">
 					</ul>
+					${PICTURE_IN_PICTURE_BUTTON}
+					${FULLSCREEN_BUTTON}
 				</div>
 			</div>`
-			base.appendChild(controlsElement.children[0])
+			this.#base.appendChild(controlsElement.children[0])
 
 			this.#controls = this.shadow.querySelector("#controls")!
 			this.#playButton = this.shadow.querySelector("#play")!
 			this.#volumeButton = this.shadow.querySelector("#volume")!
 			this.#trackButton = this.shadow.querySelector("#track")!
 			this.#trackList = this.shadow.querySelector("ul#tracklist")!
+			this.#fullscreenButton = this.shadow.querySelector("#fullscreen")!
+			this.#pipButton = this.shadow.querySelector("#picture-in-picture")!
 
 			this.#canvas.addEventListener("click", this.playPauseEventHandler)
 
 			this.#playButton.addEventListener("click", this.playPauseEventHandler)
 
 			this.#volumeButton.addEventListener("click", this.toggleMuteEventHandler)
 
+			this.#base.addEventListener("mouseenter", this.onMouseEnterHandler)
+			this.#base.addEventListener("mouseleave", this.onMouseLeaveHandler)
```

Comment:
Added this to bring up the controls when hovering over the base area when the player is in PiP mode.